packages/story/src/stories/toggle/toggle.stories.tsx

```diff
@@ -12,9 +15,43 @@ const meta: Meta<typeof Component> = {
 export default meta;
 type Story = StoryObj<typeof Component>;
 
-export const Toggle: Story = {
+export const ToggleStory: Story = {
+  name: "Toggle",
   args: {
     title: "Toggle",
     blocks: blocks,
   },
 };
+
+type ToggleProps = ToggleArgs & {
+  children?: React.ReactNode;
+};
+
+const CustomToggle = (props: ToggleProps) => {
+  return (
+    <Toggle
+      {...props}
+      customElement={
```

Comment:
p1: It is not compound pattern

you should check todo interface

Reply:
In notion.tsx, when reviewing the function Notion, if a custom configuration is present, it overwrites the components. This causes an issue where only the values of the compound's internal elements are passed as children. 

This issue causes a bug similar to #88, where components do not render correctly.

Reply:
see: https://github.com/notionpresso/react/pull/90